### ISSUE TITLE
Focus should be on help modal after selection

### ISSUE THREAD

**Issue by DanielCouzens** [epimorphics/ppd-explorer#218]
Navigation should immediately focus on help modal.

https://docs.google.com/spreadsheets/d/1YS3N5ps9pJGFOxBDi9C2DwA37WQPIVax/edit?gid=154688571#gid=154688571&range=A10:J10

**Comment by DanielCouzens** [comment]
I have combined issues 216 and 218 into one PR

please see PR https://github.com/epimorphics/ppd-explorer/pull/224

**Comment by DanielCouzens** [comment]
PR https://github.com/epimorphics/ppd-explorer/pull/224 also fixes the following issues from the report

- [x] [2.1.2: no Keyboard trap](https://docs.google.com/spreadsheets/d/1YS3N5ps9pJGFOxBDi9C2DwA37WQPIVax/edit?gid=154688571#gid=154688571&range=A11)
- [x] [2.4.11: Focus Not Obscured (Minimum)](https://docs.google.com/spreadsheets/d/1YS3N5ps9pJGFOxBDi9C2DwA37WQPIVax/edit?gid=154688571#gid=154688571&range=A13)
- [x] [top links aren't accessible easily via keyboard ](https://docs.google.com/spreadsheets/d/1YS3N5ps9pJGFOxBDi9C2DwA37WQPIVax/edit?gid=154688571#gid=154688571&range=A16)

**Comment by DanielCouzens** [comment]
@tomguilbert this is ready to check on the dev instance https://hmlr-dev-pres.epimorphics.net/app/ppd/

**Comment by tomguilbert** [comment]
Cool looks good @DanielCouzens and seems to have resolved loads of the issues in one!?
